src/Plugin.types.ts

```diff
@@ -1,7 +1,7 @@
 
 export type EnrollmentOverviewProps = {
     programId: string;
-    orgUnitId: string;
+    orgtimeUnitId: string;
```

Comment:
Revert back to orgUnitId
```suggestion
    orgUnitId: string;
```